data/build-info.json

```diff
@@ -0,0 +1,5 @@
+{
+    "os_info" : "OS_UNKNOWN",
```

Comment:
I think these could default to "unknown" with intent of being human readable.

_(see related comment on the build-info.py script)_